event_routing_backends/processors/mixins/base_transformer.py

```diff
@@ -50,11 +50,12 @@ def extract_subdict_by_keys(self, base_dict, keys):
             )
         return {key: base_dict[key] for key in set(keys).intersection(base_dict.keys())}
 
-    def find_nested(self, key):
+    def find_nested(self, event_dictionary, key):
```

Comment:
This should be static method and can rename `event_dictionary` to `source_dict`?

Reply:
Done